### PR TITLE
Feature/nanoflow

### PR DESCRIPTION
**Pull request by jseremba**
- Update icon
- Add nanoflow support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "GeoLocationForPhoneGap",
+  "version": "3.1.0",
+  "description": "",
```
Comment (McDoyen) [inline package.json]: Add the discription

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "license": "",
```
Comment (McDoyen) [inline package.json]: Add licence

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "author": "",
```
Comment (McDoyen) [inline package.json]: Remove author

File: package.json (hunk 2 above)
Comment (McDoyen) [inline package.json]: Copyright is missing

File: package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "private": true,
```
Comment (McDoyen) [inline package.json]: Remove "private": true, its a public widget

Comment (McDoyen) [inline package.json]: URLs are missing ie. repository and issues

File: src/GeoLocationForPhoneGap/GeoLocationForPhoneGap.xml [hunk 5]
```diff
@@ -2,9 +2,7 @@
...
...CAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgI
```
Comment (McDoyen) [inline src/GeoLocationForPhoneGap/GeoLocationForPhoneGap.xml]: Remove line

**Review by McDoyen — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jseremba) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (jseremba) [reply package.json]: Done

File: package.json (hunk 3 above)
Comment (jseremba) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (jseremba) [reply package.json]: Done

File: package.json (hunk 4 above)
Comment (jseremba) [reply package.json]: Done

Comment (jseremba) [reply package.json]: Done

**Pull request merged**